cluster/gce/util.sh

```diff
@@ -326,16 +326,14 @@ function upload-tars() {
   local node_binary_tar_urls=()
 
   for region in "${PREFERRED_REGION[@]}"; do
+    retention_period="1d" # https://cloud.google.com/storage/docs/bucket-lock#retention-periods
     suffix="-${region}"
-    if [[ "${suffix}" == "-us-central1" ]]; then
-      suffix=""
-    fi
     local staging_bucket="gs://kubernetes-staging-${project_hash}${suffix}"
 
     # Ensure the buckets are created
     if ! gsutil ls "${staging_bucket}" >/dev/null; then
       echo "Creating ${staging_bucket}"
-      gsutil mb -l "${region}" -p "${PROJECT}" "${staging_bucket}"
+      gsutil mb -l "${region}" -p "${PROJECT}" --retention "${retention_period}" "${staging_bucket}"
```

Comment:
I think we should add mutating the retention period on existing buckets since generally they already exist, but we can start with this.

Reply:
https://github.com/kubernetes/kubernetes/pull/116197